src/pp-annotator/PrzypisAdder.ts

```diff
@@ -108,7 +125,7 @@ export default class PrzypisAdder extends Widget {
    *
    * @param event a mouseup Event object
    */
-  onMouseUp = (event) => {
+  onMouseUp(event: JQuery.Event) {
```

Comment:
czemu zmiana z powrotem na function?

Reply:
to jest akurat work in progress jeszcze, muszę doczytać o różnicach

Reply:
główna różnica to że arrow function ma this z lexical scope, a function ma klasyczny jsowy `this`. Nie wiem na ile znasz JSa i czy mam tłumaczyć dalej czy nie :P

Reply:
to wiem, ale to nie ma znaczenia jeśli definiuje się jako część klasy w typescripcie, obie formy mają `this` zbindowany do instancji klasy. Natomiast typescript rozróżnia metody (klasyczna składnia) i properties klasy które mają typ funkcji - to są dwie różne rzeczy z punktu widzenia interfejsu. Różnią się tym, że metody klasy można override'ować, a properties nie. Więc metody klasy powinny używać raczej składni z dwukropkiem, i tak też zrefaktoruję (chyba że macie jakieś przeciwwskazania).

Więcej tutaj:
https://stackoverflow.com/questions/33400556/what-is-the-difference-between-and-on-functions-in-typescript